node/tests/src/main.rs

```diff
@@ -0,0 +1,100 @@
+//! This is a simple test for the RPC server. It checks if the server is running and can respond to.
+use std::{fs, io::Write};
+
+use clap::{Parser, Subcommand};
+use jsonrpsee::{core::client::ClientT, http_client::HttpClientBuilder, rpc_params, types::Params};
+use zksync_consensus_tools::{
+    k8s,
+    rpc::methods::{health_check::HealthCheck, RPCMethod},
+};
+
+/// Command line arguments.
+#[derive(Debug, Parser)]
+#[command(name = "tester")]
+struct TesterCLI {
+    /// Subcommand to run.
+    #[command(subcommand)]
+    command: TesterCommands,
+}
+
+/// Subcommands.
+#[derive(Subcommand, Debug)]
+enum TesterCommands {
+    /// Generate configs for the nodes.
+    GenerateConfig,
+    /// Set up the test pod.
+    StartPod,
+    /// Deploy the nodes.
+    Run,
+}
+
+/// Get the path of the node ips config file.
+/// This way we can run the test from every directory and also inside kubernetes pod.
+fn get_config_path() -> String {
+    // This way we can run the test from every directory and also inside kubernetes pod.
+    let manifest_path = std::env::var("CARGO_MANIFEST_DIR");
+    if let Ok(manifest) = manifest_path {
+        format!("{}/config.txt", manifest)
+    } else {
+        "config.txt".to_owned()
+    }
+}
+
+/// Generate a config file with the IPs of the consensus nodes in the kubernetes cluster.
+pub async fn generate_config() {
+    let client = k8s::get_client().await.unwrap();
+    let pods_ip = k8s::get_consensus_node_ips(&client).await.unwrap();
+    let config_file_path: String = get_config_path();
+    for ip in pods_ip {
+        let mut config_file = fs::OpenOptions::new()
+            .write(true)
+            .create(true)
+            .truncate(true)
+            .open(&config_file_path)
+            .unwrap();
+        config_file.write_all(ip.as_bytes()).unwrap();
```

Comment:
if it can fail - propagate the error. If it cannot, document why unwrap is safe.

Reply:
All the `unwrap` calls should be handled now. I've left the ones in the `sanity_test` function since they should function as intended for testing purposes. I'll leave a comment explaining this for clarity.